app/views/assignments/student_interface.html.erb

```diff
@@ -255,6 +229,48 @@
           <% end %>
         <% end %>
 
+
+        <% if !@group.nil? %>
```

Comment:
It seems like the indentation in the whole section is a little off.
Also, it would be nice to internationalize the strings.